packages/address/src/AddressFormatter.ts

```diff
@@ -1,20 +1,22 @@
 import {Address, FieldName, Country} from '@shopify/address-consts';
 
-import {renderLineTemplate, FIELDS_MAPPING} from './utilities';
+import {formatAddress, buildOrderedFields} from './utilities';
 import {loadCountry, loadCountries} from './loader';
 
-const FIELD_REGEXP = /({\w+})/g;
-const LINE_DELIMITER = '_';
-const DEFAULT_FORM_LAYOUT =
-  '{firstName}{lastName}_{company}_{address1}_{address2}_{city}_{country}{province}{zip}_{phone}';
-const DEFAULT_SHOW_LAYOUT =
-  '{lastName} {firstName}_{company}_{address1} {address2}_{city} {province} {zip}_{country}_{phone}';
-
-const ORDERED_COUNTRIES_CACHE: {
+let countriesCache: {
   [locale: string]: Country[];
 } = {};
 
 export default class AddressFormatter {
+
```

Comment:
This line conflicts with `prettier` rules, but is required by `lines-around-comment` rule...